dxr/plugins/xpidl/indexers.py

```diff
@@ -34,7 +34,7 @@ def idl(self):
 
     def is_interesting(self):
         # TODO: consider adding a link from generated headers back to the idl
-        return self.path.endswith('.idl')
+        return self.path.endswith('.idl') and super(FileToIndex, self).is_interesting()
```

Comment:
I picked this one since the idl filter is more restrictive and doesn't require a disk access so it makes sense to do first.